.github/workflows/nix.yml

```diff
@@ -0,0 +1,38 @@
+name: Nix builds
+
+on: [push]
```

Comment:
I don't like this on push but I'm not sure what to restrict it to. Maybe just releases

Reply:
Releases, manual and nightly?

Reply:
I can set up manual and cron for now. This project doesn't have releases yet 😬